velox/expression/tests/ExpressionFuzzer.cpp

```diff
@@ -502,15 +465,8 @@ ExpressionFuzzer::ExpressionFuzzer(
       unsupportedFunctionSignatures,
       (double)unsupportedFunctionSignatures / totalFunctionSignatures * 100);
 
-  auto functionsToTickets = getTicketsForFunctions();
-  auto getTickets = [&functionsToTickets](const std::string& funcName) {
-    auto itr = functionsToTickets.find(funcName);
-    int tickets = 1;
-    if (itr != functionsToTickets.end()) {
-      tickets = itr->second;
-    }
-    return tickets;
-  };
+  getTicketsForFunctions();
```

Comment:
These are nice refactoring, but it makes it hard to figure out the actual changes. Would it be possible to extract refactoring into a separate PR?

Reply:
I split the refactoring from the change of dereference support. Thanks for the suggestion!